app/bootstrap.php

```diff
@@ -30,6 +30,16 @@
     exit(1);
 }
 
+// PHP 8 compatibility. Define constants that are not present in PHP < 8.0
+if (!defined('PHP_VERSION_ID') || PHP_VERSION_ID < 80000) {
+    if (!defined('T_NAME_QUALIFIED')) {
+        define('T_NAME_QUALIFIED', 24001);
+    }
+    if (!defined('T_NAME_FULLY_QUALIFIED')) {
+        define('T_NAME_FULLY_QUALIFIED', 24002);
+    }
+}
+
```

Comment:
Just wondering, shouldn't we also declare `T_NAME_RELATIVE` constant?

Reply:
Perhaps no, because we don't use this constant explicitly in our code